### DISCUSSION TITLE
egzumer mod on Quansheng TG UV2 Plus?

### DISCUSSION THREAD

**Discussion by jnoake** [egzumer/uv-k5-firmware-custom#464] [General]
Is there any expectation that the egzumer Quansheng mod will work on the 10w version of their handset?
If not, is it possible to make it so (or is there some hardware impediment as with the Baofeng).

I have this on a Quansheng uv k58 and I am blown away with how it makes the radio so much better, I can't understand why the manufacturer does not deploy this as the standard firmware.

**Comment by ErikS-web** [comment]
Well as this is all hobby. Open the radio and make some photo's.

Try tio find out what kind of IC's are internally used.

**Comment by jnoake** [comment]
I see.
I don't have one, I was toying with buying one, but I don't want it if it cannot be modded to the same effect. (Already bought a 10W Baofeng that is stuck with an inadequate firmware)
I was kind of hoping someone would say 'Yes, I've done it and it works fine' or 'It should be likely since they share the same internals but for the output stage.'
Well, hope dies last.
 
  Jez.
